content/strategy-goals/strategy/code-graph/code-insights/index.md

```diff
@@ -228,9 +228,10 @@ We will explore:
 
 ### What we're not working on & why
 
-In order to focus on what drives the most user value, the Code Insights team is explicitly not working on any of the following:
+In order to focus on what drives the most user value, the Code Insights team is explicitly not working on any of the following unless new information surfaces:
 
 - Code Insights on Cloud: the current Cloud architecture strategy will bring Cloud in parity with managed and on-prem instances, so Code Insights will become available on Cloud without additional work on our part.
```

Comment:
This should be removed since we paused multitenant cloud

Reply:
You're right, thank you! Removing.